packages/pymedphys/setup.py

```diff
@@ -93,7 +93,7 @@ def get_variable_from_file(filepath, variable):
             'xlwings >= 0.15.4'
         ],
         'lint': [
-            'autopep8',
+            'black',
```

Comment:
We'll need to put this within the vscode settings also.

Reply:
Will do :)

Reply:
I didn't worry because VS Code doesn't have `autopep8` explicitly enabled in `settings.json`. I assumed `black` wouldn't need it either, and that we basically weren't enforcing *autoformatting*.

Or do you want to use `black` (with the `--check`) option for linting as well?

Reply:
I was thinking of doing the following:

https://code.visualstudio.com/docs/python/editing#_general-formatting-settings

python.formatting.provider = black

Reply:
Added :)